packages/adapter-postgres/src/index.ts

```diff
@@ -63,6 +72,36 @@ export const Adapter = createAdapter<TableTypes, FieldTypes, ConnectionConfig, C
     }
 
     const results = await db.query(select.toString());
+    const loads = context.loads;
+
+    if (loads) {
+      const loadPromises = Object.keys(loads).map(async (k) => {
+        const load = loads[k];
+        const where: { [k: string]: any[] } = {};
+        where[load.keys[1]] = results.rows.map((r) => r[load.keys[0]]);
+
+        return {
+          k, load,
+          results: await this.select(db, load.select.where(where).context)};
```

Comment:
So this will actually issue this query first? Is this bad for perf or this okay?

Reply:
@kesne if there's multiple loads, we'll do them all in parallel here since we're in an async function within the map. And it's all after the first query returns (which is necessary to build this query).